<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyrockoVLVDSourceDialog</class>
 <widget class="QDialog" name="PyrockoVLVDSourceDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>PyrockoVLVDSource Parameters</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_1">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="eastingLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoDoubleCouple.easting&lt;/span&gt;&lt;br&gt;
Coordinate of the point source.</string>
          </property>
          <property name="text">
           <string>Easting</string>
          </property>
          <property name="buddy">
           <cstring>easting</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="easting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="minimum">
           <number>-10000000</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="northingLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoDoubleCouple.northing&lt;/span&gt;&lt;br&gt;
Coordinate of the point source.</string>
          </property>
          <property name="text">
           <string>Northing</string>
          </property>
          <property name="buddy">
           <cstring>northing</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="northing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="minimum">
           <number>-10000000</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="depthLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoDoubleCouple.depth&lt;/span&gt;&lt;br&gt;
Depth of the point source.</string>
          </property>
          <property name="text">
           <string>Depth</string>
          </property>
          <property name="buddy">
           <cstring>depth</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="depth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="store_dirLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoDoubleCouple.store_dir&lt;/span&gt;
Pyrocko Green's function (GF) store directory.</string>
          </property>
          <property name="text">
           <string>GF Store</string>
          </property>
          <property name="buddy">
           <cstring>store_dir</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QFrame" name="frame_2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="store_dir">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chooseStoreDirButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>25</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="volume_changeLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoCLVDVolume.volume_change&lt;/span&gt;&lt;br&gt;
in km^3.</string>
          </property>
          <property name="text">
           <string>Volume Change</string>
          </property>
          <property name="buddy">
           <cstring>volume_change</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="volume_change">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="suffix">
           <string> km³</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="azimuthLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoCLVDVolume.strike&lt;/span&gt;&lt;br&gt;
Clockwise from north.</string>
          </property>
          <property name="text">
           <string>CLVD Azimuth</string>
          </property>
          <property name="buddy">
           <cstring>azimuth</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="azimuth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dipLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoCLVDVolume.dip&lt;/span&gt;&lt;br&gt;
Positive from horizontal.</string>
          </property>
          <property name="text">
           <string>CLVD Dip</string>
          </property>
          <property name="buddy">
           <cstring>dip</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dip">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="maximum">
           <double>90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="clvd_momentLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoCLVDVolume.clvd_moment&lt;/span&gt;
Moment in Nm, scale only.</string>
          </property>
          <property name="text">
           <string>CLVD M&lt;sub&gt;0&lt;/sub&gt; Scale</string>
          </property>
          <property name="buddy">
           <cstring>clvd_moment</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="clvd_moment">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-12.000000000000000</double>
          </property>
          <property name="maximum">
           <double>12.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="clvd_moment_exponent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="clvd_moment_exponentLabel">
          <property name="toolTip">
           <string>&lt;span style='font-family: monospace'&gt;PyrockoCLVDVolume.clvd_moment&lt;/span&gt;
Moment magnitude scale, the exponent. E.g. 5.1e&lt;b&gt;18&lt;/b&gt;.</string>
          </property>
          <property name="text">
           <string>CLVD M&lt;sub&gt;0&lt;/sub&gt; Exponent</string>
          </property>
          <property name="buddy">
           <cstring>clvd_moment</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>easting</tabstop>
  <tabstop>northing</tabstop>
  <tabstop>depth</tabstop>
  <tabstop>store_dir</tabstop>
  <tabstop>chooseStoreDirButton</tabstop>
  <tabstop>volume_change</tabstop>
  <tabstop>azimuth</tabstop>
  <tabstop>dip</tabstop>
  <tabstop>clvd_moment</tabstop>
  <tabstop>clvd_moment_exponent</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>released()</signal>
   <receiver>PyrockoVLVDSourceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
